app/scripts/lib/experiment.js

```diff
@@ -114,7 +114,7 @@ define(function (require, exports, module) {
         return;
       }
 
-      for (let experimentName in this._allExperiments) {
+      for (const experimentName in this._allExperiments) {
```

Comment:
Will this work? I was under the impression `let` had to be used inside a for expression.